### PR TITLE
Validators rewards

### PR DESCRIPTION
**Pull request by HolyGrease**

### CODE REVIEW

File: runtime/src/lib.rs [hunk 1]
```diff
@@ -135,12 +137,18 @@ pub const MILLISECS_PER_BLOCK: u64 = 60_000;
...
 
 // Time is measured by number of blocks.
 pub const MINUTES: BlockNumber = 60_000 / (MILLISECS_PER_BLOCK as BlockNumber);
 pub const HOURS: BlockNumber = MINUTES * 60;
 pub const DAYS: BlockNumber = HOURS * 24;
+// Assumse month contains 30 days
+pub const MONTHS: BlockNumber = DAYS * 30;
 
 /// Existential deposit.
 pub const EXISTENTIAL_DEPOSIT: u128 = 500;
```
Comment (avive) [inline runtime/src/lib.rs:153]: This and all other network specific params should come from genesis config file. right?

**Comment by avive** [comment]
There's still [draft] in pr title. Is it ready for review?

File: runtime/src/lib.rs (hunk 1 above)
Comment (HolyGrease) [reply runtime/src/lib.rs:153]: No, all this constants must be defined before build

Comment (avive) [reply runtime/src/lib.rs:153]: Understood. I was thinking about this and and will be great if we can have one config yaml file with all configurable values and use dev builds by default to use it with the idea to override it for testnet and mainnet. Possible?

Comment (HolyGrease) [reply runtime/src/lib.rs:153]: Definitely possible we have 2 options:
1. Throw chain spec file, but in this case we need to modify substrate pallets
2. Use some config file, but read it values while building project

**Comment by HolyGrease** [comment]
> There's still [draft] in pr title. Is it ready for review? @HolyGrease

Yes, now ready for review

File: runtime/src/lib.rs (hunk 1 above)
Comment (avive) [reply runtime/src/lib.rs:153]: I think we should do the following:
1. Have default values in code for building.
2. Override them if config file is provided for node at runtime.
3. Provide a dev/test config file for testing and run the tests with the config file.

Comment (avive) [reply runtime/src/lib.rs:153]: I think it should be in a separate pr

**Review by avive — APPROVED** [review]
LGTM

**Pull request merged**